### PR TITLE
Feature/us 033 implement swagger on history enpoints 

### PR DESCRIPTION
**Pull request by Sergio-Dom**
And documeting using swagger model or dto classes related with this US

### CODE REVIEW

File: src/main/java/com/agilethought/atsceapi/controller/HistoryController.java [hunk 1]
```diff
@@ -17,20 +17,41 @@
...
 import com.agilethought.atsceapi.service.HistoryService;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
 @RestController
 @RequestMapping(value = "/api/v1")
+@Api(value = "Api de History")
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/controller/HistoryController.java]: comments must be in english.

**Review by ivanflores15 — APPROVED** [review]

File: src/main/java/com/agilethought/atsceapi/controller/HistoryController.java (hunk 1 above)
Comment (Sergio-Dom) [reply src/main/java/com/agilethought/atsceapi/controller/HistoryController.java]: Sorry i'll fix it

**Review by luisgalindoluna — APPROVED** [review]
Looks good

**Review by EmiliaGuillen-dev — APPROVED** [review]
great

**Pull request merged**